### PR TITLE
Handle missing `@param` descriptions

### PR DESCRIPTION
**Pull request by str4d**

### CODE REVIEW

**Comment by str4d** [comment]
This panic was found in https://github.com/flipperzero-rs/flipperzero/pull/45, where I am working on transforming not-the-cleanest Doxygen comments into Rustdoc. I will likely open subsequent PRs with requests to make the parsers more tolerant, but this is the only panic preventing `doxygen-rs` from working at all.

File: src/ast/mod.rs [hunk 1]
```diff
@@ -173,24 +173,19 @@ pub fn generate_ast(input: Vec<Value>) -> ParsedDoxygen {
...
-                    };
+                    // Parse the top line into argument name, and description (if any).
+                    let mut split = content.top.splitn(2, char::is_whitespace);
+                    let arg_name = split
+                        .next()
+                        .expect("content.top should be non-empty")
+                        .to_owned();
+                    let description = split.next().map(String::from);
+
+                    // Place the description back inside the content, to preserve any sub-lists.
```
Comment (str4d) [inline src/ast/mod.rs:184]: I guessed this is what the old code was trying to do, but I can change the comment or code if I missed something.

Comment (Techie-Pi) [reply src/ast/mod.rs:184]: Yeah, I think that was the original intent :)

**Review by Techie-Pi — APPROVED** [review]
This looks great! Let's see if the tests pass and I'll release a new version :)

**Comment by Techie-Pi** [comment]
Done! Version 0.3.1 released ([crates.io/crates/doxygen-rs](https://crates.io/crates/doxygen-rs)). Thanks 👍 
PS. If you need help with the crate or find anything else, don't hesitate to contact me :)

**Pull request merged**